cypress/integration/settings-spec.js

```diff
@@ -15,7 +37,7 @@ describe('settings page', () => {
   })
 
   it('show all the items in the list', () => {
-    settingsPage.settingsList().should('have.text', 'LanguageText sizeAbout WikipediaPrivacy policyTerms of useRate the appHelp and feedbackAbout the app')
+    settingsPage.settingsList().should('have.text', settingsMenuListEnglishText.toString().replace(/,/g, ''))
```

Comment:
To make a string of all the elements in the array joined together, I recommend using `settingsMenuListEnglishText.join('')`

Reply:
fixed